tests/integration/test_batch.py

```diff
@@ -12,7 +12,7 @@ def big_dataset(dataset: Dataset):
             "row_data": IMAGE_URL,
             "external_id": "my-image"
         },
-    ] * 250)
+    ] * 2)
```

Comment:
this is a big dataset for a reason, please revert this back to 250

Reply:
I think the reason is to try to do some sort of load testing which I do not think we should do in sdk functionality test, I am discussing further with Richard